core/kernel.rbs

```diff
@@ -2782,8 +2779,7 @@ module Kernel : BasicObject
   #
   # See #respond_to?, and the example of BasicObject.
   #
-  %a{annotate:rdoc:copy:Object#respond_to_missing?}
-  private def respond_to_missing?: (Symbol, bool) -> bool
+  private def respond_to_missing?: (Symbol | String name, bool include_all) -> bool
```

Comment:
The default implementation doesn't actually check the arguments and always returns false, but the documentation says it can also take a `String`, so I added that.

Reply:
This is meant to be an abstract method but with a default implementation given.